datetimepicker/src/main/java/com/taransit/datetimepicker/dialog/DateTimePickerDialog.kt

```diff
@@ -59,6 +73,10 @@ class DateTimePickerDialog(private val cornerRadius: Float = 0f) : DialogFragmen
         super.onViewCreated(view, savedInstanceState)
 
         val cornerRadius = arguments?.getFloat(Builder.RADIUS) ?: this.cornerRadius
+        val initialHours = arguments?.getInt(Builder.HOURS) ?: this.initialHours
+        val initialMinutes = arguments?.getInt(Builder.MINUTES) ?: this.initialMinutes
+        val initialDate = arguments?.getSerializable(Builder.DATE) ?: this.initialDate
```

Comment:
V souvislosti s předchozím návrhem, tady bych zase neumožňoval nully, u žádného argumentu (včetně corner radius). Prostě všemu nastavit nějaký default a v kódu nemusíš řešit nullchecky.